Vision/Vision.bs

```diff
@@ -40,7 +40,7 @@ Markup Shorthands: markdown yes
 	* Communicate shared values and principles of the W3C community
 	* Be opinionated enough to provide a framework for making decisions,
 		particularly on controversial issues
-	* Be timeless enough to guide W3C yet flexible enough to evolve when needed
+	* Be timeless enough to remain meaningful without needing frequent revision
```

Comment:
```suggestion
	* Be timeless enough to remain meaningful yet flexible enough to evolve when needed
```

I tend to prefer emphasizing the positive (flexible, evolve) rather than negative (without, frequent) so I'd like to retain that part of the clause.

Reply:
I understand your point about positive framing, but I don't think "timeless" and "flexible to evolve" are compatible. Timeless means it doesn't need to change, because it remains relevant over time, not needing change even as things change around it.

Reply:
(Switched "meaningful" to "relevant", but don't have any ideas about the "without needing frequent revision" part that conveys the same idea.)

Reply:
Yeah, I don't think that edit you have there actually works though.